charts/datahub/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.50
+version: 0.2.51
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 appVersion: 0.8.26
```

Comment:
Is it on purpose that this wasn't bumped to 0.8.27?

Reply:
oops. sending out the pr now. this doesn't affect anything real tho. for documentation purposes